example/.gitignore

```diff
@@ -0,0 +1,2 @@
+.dart_tool/
```

Comment:
just delete this, right?

Reply:
I think it's ok to keep this around (example's ~self-sufficient).